conf/defaults.ini

```diff
@@ -182,6 +182,30 @@ cookie_samesite = lax
 # set to true if you want to allow browsers to render Grafana in a <frame>, <iframe>, <embed> or <object>. default is false.
 allow_embedding = false
 
+# Set to false if you want to disable http strict transport security (HSTS) response header.
+# This is only sent when HTTPS is enabled in this configuration.
+# HSTS tells browsers that the site should only be accessed using HTTPS.
+strict_transport_security = true
```

Comment:
I'm not sure we want to enable this by default. It's quite invasive unless the operator understands it. 

If this header is set, the browser won't allow any none HTTPS request to the same URL. Which I can see happen on internal company networks.

Reply:
Looks good otherwise!